### ISSUE TITLE
(8) Ingest of CMF metadata & files

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#242]
### Descriptive Summary

Ingest Cabrillo Music Festival recordings and metadata into the DAMS.

### Acceptance Criteria

This is what done looks like:

* [x] Metadata and files have been moved into dams_ingest
* [x] Ned has been notified
* [x] Collection ingested in production(?)
* [ ] QA of collection

### Expected Behavior 

Collection is successfully ingested.  The structure of the collection is interpreted correctly by the system (complex object and collection level metadata).

### Related Work 

This is contingent on the implementation of collection-level metadata work (#103) having been completed and #17 Get metadata and files ready for ingest.

**Comment by rmjaffe** [comment]
General question @chesleyperry @tmariamora @katedundon @NedHenry:  At this point are we trying to integrate this new material with the CMF stuff that has already been ingested as part of the CLIR pilot project?  Or are we focusing solely our work on the new content, circling back to the unpublished pilot stuff later?

**Comment by katedundon** [comment]
My understanding is that this immediate work excludes the earlier pilot project recordings (which have already been made available according to the previous grant obligations), and that the unpublished pilot stuff can be addressed at a later date as time/schedules allow. @tmariamora does that sound right?

**Comment by tmariamora** [comment]
Yes, I agree.

**Comment by rmjaffe** [comment]
@katedundon @tmariamora Thanks for confirming!

**Comment by NedHenry** [comment]
I created a new issue to investigate and fix the errors processing certain audio files. TLDR: Most of the audio files seem to have ingested successfully, but 10 or 15 of them cause processing errors when creating streamable derivate files. I need to find out what is going wrong with these, fix or work around the problem, successfully process the files, and implement code that addresses the problem automatically on future ingests. 
Processing different types of audio files can be finicky, and this is a normal sort of problem to have with this kind of collection. I think I should be able to sort it out without too much headache.
